<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogClients</class>
 <widget class="QDialog" name="DialogClients">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenu{
background-color:   rgb(50, 50, 50);
color: white;
font-family: &quot;Georgia&quot;;
font-weight: bold;
font-size: 14px;
}
QMenu::item:disabled{
background-color: rgb(147, 147, 147);
color: rgb(90, 90, 90);
}

QMenu::item:disabled:selected{
background-color:  rgb(147, 147, 147);
}



QMenu::item:selected  {
background-color: rgb(198, 66, 0);
}

QMenu::separator {
height: 0px;
}


#DialogClients{
background-color: rgb(51, 51, 51);
}

QPushButton{
background-color: rgb(80, 80, 80);
border-radius: 8px;
padding:8px  5px;
text-align: center;
font-family: &quot;Georgia&quot;;
font-weight: bold;
font-size: 14px;
color: rgb(232, 232, 232);
}

QPushButton:hover{
background-color: rgb(198, 66, 0);
}

QLabel{
text-align: center;
font-family: &quot;Georgia&quot;;
font-weight: bold;
font-size: 14px;
color: rgb(232, 232, 232);
}

QLineEdit{
background-color: rgb(51, 51, 51);
border:none;
border-bottom: 2px solid  rgba(198, 66, 0,100);
font-family: &quot;Georgia&quot;;
font-size: 14px;
color: rgb(232, 232, 232);
}

/*rgb(198, 66, 0);*/</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>8</number>
      </property>
      <property name="verticalSpacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="IpAddressLine">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter ip-address user-host&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="LoginLine"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="PasswordLine">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="CreateLobby">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Create lobby</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ExitButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
